db/memtable_list.cc

```diff
@@ -726,6 +741,15 @@ Status InstallMemtableAtomicFlushResults(
     assert(0 == num_entries);
   }
 
+  std::unique_ptr<VersionEdit> wal_deletions;
```

Comment:
@riversand963  `InstallMemtableAtomicFlushResults` didn't handle 2pc mode, is 2pc not supported for atomic flush or did I miss anything?